lib/tasks/tests_data.rake

```diff
@@ -162,5 +162,8 @@ namespace :db do
                             author: 'Gurra',
                             category: 'Visor',
                             content: 'Sverige är Sverige och Sverige är bra!')
+
+    # Notifications
+    Rpush::Gcm::App.create!(name: :firebase, auth_key: :test)
```

Comment:
You can use `find_or_create_by!` instead of `create!` in this file. This will save us from getting duplicates if the script is run several times.